src/test/java/com/griddynamics/qa/AbstractTest.java

```diff
@@ -0,0 +1,34 @@
+package com.griddynamics.qa;
+
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.AfterSuite;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.BeforeSuite;
+import ru.yandex.qatools.allure.annotations.Step;
+
+public class AbstractTest {
```

Comment:
In fact the class isn't an abstract one, so I would prefer to go with "common" or "base" prefix.